hips/draw/simple.py

```diff
@@ -178,11 +179,48 @@ def draw_tiles(self) -> np.ndarray:
 
         return image
 
+    def draw_hips_tile_grid(self) -> None:
```

Comment:
@cdeil Then `_draw_sky_image` function would only contain the line `ax.imshow(image, origin='lower')`?